numpy/array_api/_typing.py

```diff
@@ -15,10 +15,12 @@
     "PyCapsule",
 ]
 
-from typing import Any, Literal, Sequence, Type, Union
+import sys
+from typing import Any, Literal, Sequence, Type, Union, TYPE_CHECKING, TypeVar
 
-from . import (
-    Array,
+from ._array_object import Array
```

Comment:
`Array` does not live in the main `np.array_api` namespace, so can't import it from there.

Reply:
Thanks for fixing it!